rdr_service/genomic/genomic_job_controller.py

```diff
@@ -1941,6 +1941,31 @@ def check_appointments_gror_changed(self):
         else:
             self.job_result = GenomicSubProcessResult.NO_RESULTS
 
+    def check_gcr_14day_escalation(self):
+        """Alerts color when participant with hdr positive result does not have a scheduled or completed appointment"""
+        notified_dao = GenomicGCROutreachEscalationNotifiedDao()
+        notification_email_addresses = config.getSettingList(config.GENOMIC_GCR_ESCALATION_EMAILS, default=None)
+        escalate_participants = self.report_state_dao.get_hdr_result_positive_no_appointment()
+        if notification_email_addresses and escalate_participants:
+            notified_participants = []
+            for pid in escalate_participants:
+                EmailService.send_email(
+                    Email(
+                        recipients=notification_email_addresses,
+                        subject='GCR Outreach 14 Day Escalation',
+                        plain_text_content=str(pid)
+                    )
+                )
```

Comment:
Are we sending one for each participant? What if there are hundreds? Might be better to double-check since we don't want to spam Color with this.

Reply:
Yes, sending one email for each PID is a requirement from the ticket.